flake.nix

```diff
@@ -97,8 +97,14 @@
           k3sVMs = import ./modules/k3s { inherit system inputs pkgs username; };
 
           # k3s VM config
-          k3sVMConfig = k3sVMs.buildConfig {
+          k3sServerVMConfig = k3sVMs.buildConfig {
             kube_vip = "192.168.69.20";
+            k3s_role = "server";
+            n = 3;
+          };
+          k3sAgentVMConfig = k3sVMs.buildConfig {
+            kube_vip = "192.168.69.20";
+            k3s_role = "agent";
```

Comment:
Seems to work well, might be able to condense at some point, but probably want to move away from k3s at that point.